test/Graphics/RenderWindow.test.cpp

```diff
@@ -1,8 +1,60 @@
+#include <SFML/Graphics/Image.hpp>
 #include <SFML/Graphics/RenderWindow.hpp>
+#include <SFML/Graphics/Texture.hpp>
+#include <SFML/System/String.hpp>
+#include <SFML/Window/VideoMode.hpp>
 
+#include <doctest/doctest.h>
+
+#include <GraphicsUtil.hpp>
 #include <type_traits>
 
 static_assert(!std::is_copy_constructible_v<sf::RenderWindow>);
 static_assert(!std::is_copy_assignable_v<sf::RenderWindow>);
 static_assert(!std::is_nothrow_move_constructible_v<sf::RenderWindow>);
 static_assert(!std::is_nothrow_move_assignable_v<sf::RenderWindow>);
+
+TEST_CASE("[Graphics] sf::RenderWindow" * doctest::skip(skipDisplayTests))
+{
+    SUBCASE("Construction")
+    {
+        sf::RenderWindow window(sf::VideoMode(sf::Vector2u({256, 256}), 24),
```

Comment:
```suggestion
        const sf::RenderWindow window(sf::VideoMode(sf::Vector2u(256, 256), 24),
```
I won't comment this everywhere, but please put `const` as many local variables used in the tests as possible. It's important to ensure that member functions that should be marked `const` actually are.